charts/opensrp-web/values.yaml

```diff
@@ -88,13 +88,14 @@ containerEnvironmentVariables:
   EXPRESS_OPENSRP_OAUTH_STATE: "{{ .Values.express.opensrpOauthState }}"
   EXPRESS_SESSION_PATH: "{{ .Values.express.sessionPath }}"
   EXPRESS_SESSION_LOGIN_URL: "{{ .Values.express.sessionLoginUrl }}"
-  EXPRESS_FRONTEND_LOGOUT_URL: "{{ .Values.express.sessionFileStorePath }}"
+  EXPRESS_FRONTEND_LOGOUT_URL: "/tmp/{{ .Values.sharedVars.releaseNameAndNamespace }}-express-sessions"
```

Comment:
this value doesn't seem correct

Reply:
@KipSigei its the new var in the default values